scripts/build/Platform/Windows/build_config.json

```diff
@@ -207,7 +207,7 @@
     "PARAMETERS": {
       "CONFIGURATION": "profile",
       "OUTPUT_DIRECTORY": "build\\windows",
-      "CMAKE_OPTIONS": "-DCMAKE_SYSTEM_VERSION=10.0 -DO3DE_TEST_IMPACT_NATIVE_TEST_TARGETS_ENABLED=FALSE -DO3DE_TEST_IMPACT_PYTHON_TEST_TARGETS_ENABLED=TRUE",
+      "CMAKE_OPTIONS": "-DCMAKE_SYSTEM_VERSION=10.0 -DO3DE_TEST_IMPACT_NATIVE_TEST_TARGETS_ENABLED=TRUE -DO3DE_TEST_IMPACT_PYTHON_TEST_TARGETS_ENABLED=TRUE",
```

Comment:
question: Is TIAF windows only? What about Linux?

Reply:
TIAF is Windows only for now. It would be relatively straightforward to support Linux, but it would require implementing the necessary platform and toolchain specific support as there is no common test coverage tool on both platforms or compiler support for coverage between all supported compilers on both platforms, hence the decision to focus on Windows to demonstrate the capabilities before considering Linux support.